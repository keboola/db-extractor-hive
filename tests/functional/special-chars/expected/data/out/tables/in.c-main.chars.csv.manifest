{
    "destination": "in.c-main.chars",
    "incremental": false,
    "metadata": [
        {
            "key": "KBC.name",
            "value": "chars"
        },
        {
            "key": "KBC.sanitizedName",
            "value": "chars"
        },
        {
            "key": "KBC.schema",
            "value": "default"
        },
        {
            "key": "KBC.catalog",
            "value": "HIVE"
        },
        {
            "key": "KBC.type",
            "value": "table"
        }
    ],
    "columns": [
        "text1",
        "text2"
    ],
    "column_metadata": {
        "text1": [
            {
                "key": "KBC.datatype.nullable",
                "value": true
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "STRING"
            },
            {
                "key": "KBC.datatype.default",
                "value": ""
            },
            {
                "key": "KBC.sourceName",
                "value": "text1"
            },
            {
                "key": "KBC.sanitizedName",
                "value": "text1"
            },
            {
                "key": "KBC.uniqueKey",
                "value": false
            },
            {
                "key": "KBC.ordinalPosition",
                "value": 1
            }
        ],
        "text2": [
            {
                "key": "KBC.datatype.nullable",
                "value": true
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "STRING"
            },
            {
                "key": "KBC.datatype.default",
                "value": ""
            },
            {
                "key": "KBC.sourceName",
                "value": "text2"
            },
            {
                "key": "KBC.sanitizedName",
                "value": "text2"
            },
            {
                "key": "KBC.uniqueKey",
                "value": false
            },
            {
                "key": "KBC.ordinalPosition",
                "value": 2
            }
        ]
    }
}