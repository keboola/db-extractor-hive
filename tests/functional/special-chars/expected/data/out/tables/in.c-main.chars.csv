"Přeji hezký pátek","!@#$%^&*()_"
"úěščřžýáíé+=","Τη γλώσσα μου έδωσαν ελληνική"
